commands/build/lib/build.js

```diff
@@ -275,13 +318,19 @@ async function build(argv = {}) {
   if (buildConfig.watch) {
     return watch(buildConfig);
   }
-  await minified(buildConfig);
 
+  // Standalone
+  await umd(buildConfig);
+  await esm(buildConfig);
+
+  // Core
   if (argv.core) {
     const core = path.resolve(process.cwd(), argv.core);
     await esm(buildConfig, core);
+    await systemjs(buildConfig, core);
   }
-  return esm(buildConfig);
+
+  return Promise.resolve();
```

Comment:
there is no need to create an Promise here
```suggestion
  return undefined;
```
or remove the return (if it is not needed by lint rules)

Reply:
Nice one. Was a bit too eager to not break backward compatibility here.